scripts/naruon_compose.sh

```
@@ -17,10 +19,95 @@ EOF
+    fi
+  fi
+fi
+
+case "${llm_runtime}" in
+  ollama)
+    compose_files=()
+    ;;
+  mlx|llama.cpp)
+    if [ "${llm_runtime}" = "mlx" ]; then
+      host_llm_base_url="${NARUON_MLX_BASE_URL:-http://host.docker.internal:8080/v1}"
+      export NARUON_HOST_LLM_BASE_URL="$(container_host_url "${host_llm_base_url}")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)